apps/ui/src/components/Ui/BadgeNetwork.vue

```diff
@@ -27,12 +26,14 @@ const currentNetwork = computed(() => {
 <template>
   <div class="relative">
     <img
-      :src="(currentNetwork && getUrl(currentNetwork.avatar)) ?? undefined"
-      :title="(currentNetwork && currentNetwork.name) ?? undefined"
+      v-if="currentNetwork"
+      :src="getUrl(currentNetwork.avatar) ?? undefined"
```

Comment:
Missed it before, but now that `currentNetwork` network exist for sure, means that we don't need the fallback `?? xxx` below

Reply:
looks like `getUrl` can still return `null`, so using fallback here but removed for others